src/components/filterBar.js

```diff
@@ -72,25 +81,36 @@ const FilterBar = ({
         onDateChangeUntil(dateUntil);
     }
 
+    const onSetSearchLimit = (event) => {
+        setSearchLimit(event.target.value);
+    }
+
+    const onResetFilters = () => {
+        setDateFilterValue("");
+        onDateChangeFrom(null);
+        onDateChangeUntil(new Date());
+        setSearchLimit(100);
```

Comment:
could be nice to make a shared constant for the default search limit, because i think it's also defined in the initialState of the reducer